idea-plugin/build.gradle

```diff
@@ -0,0 +1,82 @@
+apply plugin: 'java'
+apply plugin: 'maven-publish'
+apply plugin: 'org.jetbrains.intellij'
+apply plugin: 'com.palantir.external-publish-jar'
+
+intellij {
+    pluginName = "gradle-jdks"
+    updateSinceUntilBuild = true
+    version = "2024.1"
+    plugins = ['java', 'org.jetbrains.plugins.gradle']
+}
+
+def buildPlugin = tasks.named("buildPlugin")
+
+patchPluginXml {
+    pluginDescription = "Configures the Gradle JDKs for the Toolchains used for a project and for the Gradle Daemon."
+    version = project.version
+    sinceBuild = '232' // TODO: test against this version of IntelliJ to ensure no regressions
```

Comment:
Is this just copied from PJF? Not sure how we actually work out which build is the earliest we support. I know the plugin runs through compatibility tests when you publish it though.

Reply:
> Is this just copied from PJF?
Yes, I tested this version via `runIde` though by changing the `intellij.version` to 2023.2
I guess most people are running on the latest so I can change this to match `2024.1`

Reply:
We probably want to keep it relatively old so people don't need to upgrade.

> I guess most people are running on the latest so I can change this to match 2024.1

I want this to be true but I keep seeing people on really, really old versions lol